src/commands/dao_cmds/utils/web3-utils.js

```diff
@@ -0,0 +1,27 @@
+import Web3 from 'web3'
+
+// Check address equality without checksums
+export function addressesEqual(first, second) {
+  first = first && first.toLowerCase()
+  second = second && second.toLowerCase()
+  return first === second
+}
+
+// Cache web3 instances used in the app
+const cache = new WeakMap()
+export function getWeb3(provider) {
+  if (cache.has(provider)) {
+    return cache.get(provider)
+  }
+  const web3 = new Web3(provider)
+  cache.set(provider, web3)
+  return web3
+}
+
+export function weiToEther(wei) {
```

Comment:
These functions are not used either, and it is better to use `web3.fromWei` as they handle BigNumber